src/features/moreTab/more/MoreListItem.tsx

```diff
@@ -60,6 +72,11 @@ const MoreListItem = ({
     [],
   )
 
+  let textColor = 'primaryText'
```

Comment:
This needs a type to fix lint errors:

` let textColor: Colors = 'primaryText'`